### PR TITLE
updated README required port 3000

### PR DESCRIPTION
**Pull request by tonysorensen**
# Pull Request Template

## Description
Add a summary that address the following:
 - motivation/purpose for the feature 
     Advise developers of okta port requirement.
 - What were those changes
     Updated the README to advise that okta requires the front end to be run on port 3000 when working locally
 - What issues were fixed
     n/a
 - Provide the link to the user story it is referencing in Trello board or other supporting document
   n/a

## Commits Checklist
- [ x] commits have clear title names 
- [ x] commit has descriptive message of what has changed 
- [ x] more small commits than less big commits 

## Code Checklist 
- [ x] code is formatted appropriately 
- [ x] code meets DRY standards
- [x ] no dead code (check for logs and print statements)  
- [ x] no TODOS instead necessary comments 
- [ x] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries   

### PRs needs to be reviewed by at least 2 team members
 - TPL is the 3rd reviewer 
 - Lots of comments and suggestions, please! 

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by SethMacP — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -5,15 +5,20 @@
 [<img src="https://avatars2.githubusercontent.com/t/4044276?s=280&v=4" width="200" />](https://avatars2.githubusercontent.com/t/4044276?s=280&v=4)
 
 Story Squad turns “reluctant readers” into authors and illustrators through a collaborative world-building game.
+
 <!-- Todo! Update this -->
+
 You can find the deployed project at: [AWS Amplify](https://a.storysquad.dev/)
```
[the remark this answers is not in the document]
Comment (tonysorensen) [reply README.md:12]: That's a good catch. When I first ran into the problem I was just focused on getting things started.

**Review by chukalicious — DISMISSED** [review]
Thank you for the warning on this Tony!

**Review by chukalicious — APPROVED** [review]
Thank you Tony, great job!

**Review by jproctor-rebecca — APPROVED** [review]
Thank you, Tony! This was important for incoming teams.

**Pull request merged**